### PR TITLE
Fix grammar erros, make english sound more native, host images on GitHub itself

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
oh no i broke git

File: README.md [hunk 1]
```diff
@@ -50,23 +50,23 @@ I have a lot of interests in programming, but here are some things I love a lot
...
+- **EzChRoot:** chroots are essential when one is making a build system (You know, you don't want to delete /etc/ on your main system on accident), so so I made EzChRoot, since all where bash scripts, and allowed very little reusability or configuration I wrote my own tool on Golang, sadly the code isn't very good and it is broken on Big Sur.
 
-- **trm:** This idea is simple in fact it was suggested by a friend, it moves the file to trash instead of deleting it, like rm does, and you might wonder "Why not just make an alias? to mv" and you would be 100% right, I only made it for fun and a suggestion.
+- **trm:** This project is simple, in fact it was suggested by a "friend", its basically `rm` but instead of deleting files it moves them to the trash, and you might wonder "Why not just make an alias to mv?" and you would be 100% right, I only made it for fun and a suggestion.
```
Comment (DiegoMagdaleno) [inline README.md]: Revert this back to friend, aka delete the quotes

**Review by DiegoMagdaleno — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
lol ok

**Comment by rodent5** [comment]
do I keep the question mark change in "why not just use an alias"

**Pull request merged**